### PR TITLE
Python localstack demo

### PR DESCRIPTION
**Pull request by gokhan721**

### CODE REVIEW

**Review by erdogan-yesil — APPROVED** [review]

File: tests/conftest.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# from pathlib import Path
```
Comment (serkan-ozal) [inline tests/conftest.py:1]: Seems that this file is left over

File: thundra_demo_localstack/README.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
+<!--
```
Comment (serkan-ozal) [inline thundra_demo_localstack/README.md]: No need to keep this readme file as it is relevant

File: tests/conftest.py (hunk 1 above)
Comment (gokhan721) [reply tests/conftest.py:1]: I just intentionally left this file. These are another configuration styles for setting Thundra Agent. If someone would like to set Thundra by one of these method, uncommenting one of these configuration style and setting appropriate values is only things they have to do.

File: thundra_demo_localstack/README.md (hunk 2 above)
Comment (serkan-ozal) [reply thundra_demo_localstack/README.md]: @gokhan721 what about this one?

Comment (gokhan721) [reply thundra_demo_localstack/README.md]: This readme file is automatically generated by Serverless. I have removed it.

**Review by serkan-ozal — APPROVED** [review]

**Pull request merged**